### DISCUSSION TITLE
Question on the need of showing fiber usage fraction

### DISCUSSION THREAD

**Discussion by monodera** [Subaru-PFS/spt_target_uploader#160] [Ideas]
@takagi-naoj raised a question about the reason why we need to display fiber usage fractions. He wondered whether it affects the reviewers' scores when reviewing proposals.

Any ideas?

**Comment by mihoishigaki** [comment]
In my opinion, we don't have to show the fiber usage fractions, as far as we operate with fiber sharing. Reviewers will be asked to evaluate whether the program makes good use of Subaru/instrument. However, the fiber usage fraction alone would not be a good/objective figure to evaluate this and could be misused.
